spec/services/evss_claim_service_spec.rb

```diff
@@ -48,12 +48,12 @@
     it 'enqueues a job' do
       expect do
         subject.upload_document(document)
-      end.to change(EVSS::DocumentUpload.jobs, :size).by(1)
+      end.to change(Workflow::Runner.jobs, :size).by(1)
     end
 
-    it 'updates document with sanitized filename' do
+    xit 'updates document with sanitized filename' do
```

Comment:
Sorta? The behavior itself was a side effect of something carrier wave did, but I think we need to determine if it still needs to happen.